tests/backends/test_native_backend_against_coincurve.py

```diff
@@ -157,3 +147,62 @@ def test_coincurve_to_native_invalid_signatures(message_hash,
     public_key_b = backend_b.ecdsa_recover(message_hash, signature_a)
 
     assert public_key_b == public_key_a
+
+
+@given(
+    private_key_bytes=private_key_st,
+    direction=st.one_of(
+        st.just('coincurve-to-native'),
```

Comment:
What's the point of reversing direction on this? Looks like copy-pasta, to me.

Reply:
Whoops, you're right, there's no point. Fixed.